src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, isMarried, sex, partnerName } = person;
```

Comment:
Ensure that all properties destructured from `person` are available. If `partnerName` is undefined, it could lead to unexpected behavior or errors when rendering `partnerSex`.